02-microarray/gene-id-annotation_microarray_01_ensembl.Rmd

```diff
@@ -283,27 +289,26 @@ head(mapped_df)
 ```
 
 We can see that our data frame has a new column `Symbol`.
-Let's get a summary of the gene symbols returned in the `Symbol` column of our mapped data frame.
+Let's get a summary of the gene symbols in the `Symbol` column of our mapped data frame.
 
 ```{r}
-# We can use the `summary()` function to get a better idea of the distribution of symbols in the `Symbol` column
-summary(mapped_df$Symbol)
+# Use the `summary()` function to show the distribution of Symbol values
+# We need to use `as.factor()` here to get the counts of unique values
```

Comment:
Probably should add a short comment explaining the `maxsum` bit.

Reply:
Done!